autoscheduler/frontend/src/components/SchedulingPage/SchedulingPage.tsx

```diff
@@ -1,22 +1,24 @@
 import * as React from 'react';
-import { useDispatch } from 'react-redux';
-import { RouteComponentProps } from '@reach/router';
+import { useDispatch, useSelector } from 'react-redux';
+import { RouteComponentProps, navigate } from '@reach/router';
 import Schedule from './Schedule/Schedule';
 import * as styles from './SchedulingPage.css';
 import ConfigureCard from './ConfigureCard/ConfigureCard';
 import SchedulePreview from './SchedulePreview/SchedulePreview';
 import CourseSelectColumn from './CourseSelectColumn/CourseSelectColumn';
 import setTerm from '../../redux/actions/term';
+import { RootState } from '../../redux/reducer';
```

Comment:
Make sure to fix these linting errors

Reply:
Wait would the 3 time call also apply to the fetch last term? Cause the navigate call is just directly inside that promise.

Reply:
You can do `waitFor(() => expect(navSpy).toHaveBeenCalledTimes(50)` and it'll say how many were called (if you do less than the number of times it's been called it will have at some point been that value so it'll pass and not show you how many there actually were)

Reply:
Cool thanks!